inst/stan/functions/discretised_logit_hazard.stan

```diff
@@ -210,14 +225,13 @@ vector lprob_to_uniform_double_censored_log_hazard(vector lprob, vector lcdf,
  * distributions to logit hazards.
  * 
  * @param lhaz Vector of log hazards.
- *
- * @param n Number of hazards.
  * 
  * @return Vector of logit hazards.
  * 
  * @note Final transformation step in `discretised_logit_hazard`.
  */
-vector log_hazard_to_logit_hazard(vector lhaz, int n) {
+vector log_hazard_to_logit_hazard(vector lhaz) {
+  int n = num_elements(lhaz);
```

Comment:
Note that I made `log_hazard_to_logit_hazard` more standalone by removing the `n `argument. I think this is justified here because this is essentially a vectorized operation, and the individual elements of the vector do not interact.